tests/integration_tests/test_chainermn.py

```diff
@@ -21,7 +21,7 @@
 from optuna.storages import BaseStorage
 from optuna.storages import InMemoryStorage
 from optuna.storages import RDBStorage
-from optuna.testing.integration import DeterministicPruner
+from optuna.testing.pruner import DeterministicPruner
 from optuna.testing.sampler import DeterministicRelativeSampler
```

Comment:
[Notes] At first glance, the module might be inconsistent with `optuna.pruners`, but I found that the other modules in `optuna.testing` including `sampler`, `distribution` and `storage` are also singular.